### ISSUE TITLE
Can't update node settings: Table 'pterodactyl.settings' doesn't exist

### ISSUE THREAD

**Issue by Tommsy64** [pterodactyl/panel#1650]
* Version of Panel: v0.7.14 (php version `PHP 7.3.7 (cli) (built: Jul  8 2019 09:20:53) ( NTS ) `)
* Server's OS: Ubuntu 18.04.2 LTS

On a fresh install of the panel, I made a location and created one node at that location. When I try to change the settings of the node there is an internal server error.

access.error.log is empty

access.log

```
src:"***.***.***.***" time:"24/Jul/2019:06:21:05 +0000" req:"POST /admin/nodes/view/3/settings HTTP/1.1" status:"500" agent:"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.142 Safari/537.36" for:"***.***.***.***"
```

The  laravel.log had these two lines in it; however, they do not correspond to when the internal server error occurs when pressing the "Save Changes" button.

```
[2019-07-24 01:09:11] production.NOTICE: A query exception was encountered while trying to load settings from the database: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'pterodactyl.settings' doesn't exist (SQL: select * from `settings`)
[2019-07-24 01:09:17] production.NOTICE: A query exception was encountered while trying to load settings from the database: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'pterodactyl.settings' doesn't exist (SQL: select * from `settings`)
```

Running `php artisan migrate` returns "Nothing to migrate."

**To Reproduce**
1. Fresh install of panel.
2. Add a location.
3. Add a node.
4. Press "Save Changes".
5. Get internal server error regardless of whether or not any settings were changed.

**Expected behavior**
Successfully change settings of existing node.

**Comment by MatthewSH** [comment]
Have you talked to anyone in support channels yet before reporting this as an issue? https://discord.gg/pterodactyl

**Comment by Tommsy64** [comment]
I've restarted the panel and database several times trying to get this to work. I ran `php artisan migrate --seed`, restarted panel and DB. This did not work. Finally, I added `APP_REPORT_ALL_EXCEPTIONS=true` to the .env to see if maybe I could see a stacktrace in the logs, restarted the panel, and now the error is no longer occurring. After removing `APP_REPORT_ALL_EXCEPTIONS=true`, it is still working.

I do not know exactly what fixed it.
